python/tchannel/tornado/peer.py

```diff
@@ -451,6 +478,13 @@ def send(self, arg1, arg2, arg3, traceflag=False, headers=None):
         arg1.close()
         endpoint = yield read_full(arg1)
 
+        is_stream = False
+        if (isinstance(arg2, InMemStream) and
+                isinstance(arg3, InMemStream) and
+                arg2.state == StreamState.completed and
+                arg3.state == StreamState.completed):
+            is_stream = False
```

Comment:
What is going on here? The condition is really complicated and doesn't actually change `is_stream`.